src/AdminGeneratorServiceProvider.php

```diff
@@ -27,6 +27,7 @@ public function boot()
             Generate\StoreRequest::class,
             Generate\UpdateRequest::class,
             Generate\DestroyRequest::class,
+            Generate\ImpersonalLoginRequest::class,
```

Comment:
It generates the request class - I think it's ok, or?

Reply:
Ok, it just unique class for admin user, so it makes no sense to generate it otherwise, how ever with specific request approach, we do not have other choice I guess.